uqcsbot/scripts/dice.py

```diff
@@ -0,0 +1,31 @@
+from random import choice
+from uqcsbot import bot, Command
+
+def is_number(message):
+    '''
+    Tries to coerce the message to an integer. If successful, return True-- else
+    returns False.
+    '''
+    try:
+        int(message)
+    except:
+        return False
+    return True
+
+
+@bot.on_command("dice")
+def handle_dice(command: Command):
+    '''
+    `!dice [number]` - Rolls 1 or more six sided dice (d6).
+    '''
+    if command.has_arg() and is_number(command.arg):
```

Comment:
command.arg.isnumeric()

Reply:
`command.has_arg()` needed to short circuit in the case of no arguments.  
Fixed.